webpack/scenes/Packages/PackagesActions.js

```diff
@@ -0,0 +1,34 @@
+import api, { orgId } from '../../services/api';
+import {
+  PACKAGES_REQUEST,
+  PACKAGES_SUCCESS,
+  PACKAGES_FAILURE,
+} from './PackagesConstants';
+import { getResponseError } from '../../move_to_foreman/common/helpers.js';
+import { propsToSnakeCase } from '../../services/index';
+
+export const getPackages = (extendedParams = {}) => (dispatch) => {
+  dispatch({ type: PACKAGES_REQUEST });
+
+  const params = {
+    ...{ organization_id: orgId },
+    ...propsToSnakeCase(extendedParams),
+  };
+
+  return api
+    .get(`/packages`, {}, params)
+    .then(({ data }) => {
+      dispatch({
+        type: PACKAGES_SUCCESS,
```

Comment:
you should consider using https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/redux/actions/common/index.js#L3 as it offer a standardize way to dispatch api related redux actions